tensorflow_addons/utils/test_utils.py

```diff
@@ -26,8 +26,10 @@
 from tensorflow_addons.utils import resource_loader
 
 # TODO: copy the layer_test implementation in Addons.
-from tensorflow.python.keras.testing_utils import layer_test  # noqa: F401
-
+if tf.__version__[:3] > "2.5":
+    from keras.testing_utils import layer_test  # noqa: F401
+else:
+    from tensorflow.python.keras.testing_utils import layer_test  # noqa: F401
```

Comment:
@qlzh727 As we are near to the release do we sill need to maintain this conditional import?

Reply:
yea, please keep those conditional imports here. We will check with API owner about exposing certain API to internal name space.